### ISSUE TITLE
There was an error when I ran the program according to your steps. Can you help me?Thank you very much!

### ISSUE THREAD

**Issue by Xjunjiang** [wuhuikai/FastFCN#20]
Using poly LR Scheduler!
Starting Epoch: 0
Total Epoches: 120
  0%|                                                                                                                                                                                                             | 0/1263 [00:00<?, ?it/s]
=>Epoches 0, learning rate = 0.0100,                 previous best = 0.0000
Segmentation fault (core dumped)

**Comment by wuhuikai** [comment]
Please provide the details of the running environment. But first please try to run it again :)

**Comment by wuhuikai** [comment]
@Xjunjiang The code in branch `latest` is now ready, which uses official `SyncBatchNorm` and can run on all OS with `PyTorch>=1.1.0`.
